### PR TITLE
[DOC] Updating OS documentation

### PR DESCRIPTION
**Pull request by bastienpaulfr**

### CODE REVIEW

**Comment by bastienpaulfr** [comment]
Pour la preview : http://wiki.gitlab-pages.coppernic.local/copperdoc/#/os/policy

File: os/policy.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Support
+-------
+
+Coppernic is giving support for developing on devices during the whole lifetime of the device.
+We are giving support for OS bugfixes during  ???
+
+System Applications
+-------------------
+
+OS signature keys are private to Coppernic and are given to nobody. No exceptions.
```
Comment (mreynier) [inline os/policy.md:19]: `OS signature keys are private to Coppernic and are not provided for security reason.`

File: os/policy.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+If you want to develop system applications, then you need to send your application to our [support center](http://support.coppernic.fr) for signature.
+
+Kiosk applications
+------------------
+
+We can give some pieces of advice to choose the kiok application that's right for you. We do not have our own product.
```
Comment (mreynier) [inline os/policy.md:26]: `We can give some pieces of advice to choose the kiosk application that's right for you. We do not have our own product.`

File: os/policy.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+There are several ways to keep your applications up to date on your Coppernic devices
+
+- **ADB** : Very handful at development stage. This tool from google allows you to do many things with your device. https://developer.android.com/studio/command-line/adb
+
+- [F-Droid](fdroid.md) : Your application can be deployed on F-Droid and available on all your devices for an update
+
+- **EMM** : You can deploy your app wit an EMM such as MobiControl.
+
+Both F-Droid and MobiControl can updat your application even when device is kiosked.
```
Comment (mreynier) [inline os/policy.md:39]: `Both F-Droid and MobiControl can update your application even when device is kiosked.`

**Pull request closed without merging**